jx_sqlite/query_table.py

```diff
@@ -124,7 +134,7 @@ def query(self, query):
 
         result = self.db.query(command)
 
-        column_names = query.edges.name + query.groupby.name + listwrap(query.select).name
+        column_names = listwrap(query.edges).name + listwrap(query.groupby).name + listwrap(query.select).name
```

Comment:
the `query.edges` should always be a FlatList, so you should not need to do this listwrap() here